common/src/main/java/org/apache/drill/common/expression/SchemaPath.java

```diff
@@ -54,15 +50,16 @@ public static SchemaPath getSimplePath(String name) {
   }
 
   public static SchemaPath getCompoundPath(String... strings) {
```

Comment:
Given the concern expressed in the commit message, do you think a Javadoc should be provided?

Or were you simply referencing that the logic inside the method?